src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const splited = sourceString.split(';');
```

Comment:
The variable name 'splited' is a grammatical error and does not clearly describe the contents. Consider renaming it to 'styleDeclarations' or 'styleParts' for clarity and correct grammar.